app/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
         minSdkVersion 17
         targetSdkVersion 28
         versionCode 18
```

Comment:
If you increase the Version name, can you please also increase the version code integer by 1? 
Thank you

Reply:
Good.